tests/unit/objects/test_project_merge_request_approvals.py

```diff
@@ -169,6 +298,14 @@ def test_project_approval_manager_update_method_post(project):
     assert approvals._update_method is UpdateMethod.POST
 
 
+def test_list_project_approval_rules(project, resp_prj_approval_rules):
+    approval_rules = project.approvalrules.list()
+    assert len(approval_rules) == 1
+    assert approval_rules[0].name == approval_rule_name
+    assert approval_rules[0].id == approval_rule_id
+    repr(approval_rules)  # ensure that `repr()` doesn't raise an exception
```

Comment:
Question - if we're adding the test, shouldn't we be able to assert on the string returned by the repr here instead of having a comment?

Reply:
we certainly can, I had just mirrored the similar test being done in the function below this one. I'll update mine and happy to update the other if desired.

Reply:
updated